torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1242,8 +1242,10 @@ XLATensorPtr full_symint(at::SymIntArrayRef sym_size,
                          const torch::lazy::BackendDevice& device,
                          at::ScalarType scalar_type) {
   XLA_CHECK(std::all_of(sym_size.begin(), sym_size.end(), [](at::SymInt dim) {
-    if (!dim.is_symbolic()) {
-      return dim >= 0;
+    // TODO: It should be OK to perform this test on symbolic ints too, not
```

Comment:
One reason I can think of is that if we do something like `symint>0` then PyTorch/XLA will trigger a compilation and execution which in turn cause a graph break.

Reply:
But even for a dynamically sized tensor, the sizes should be guaranteed to be >= 0.